<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>539</width>
    <height>428</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Resty</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lbl_rest_message_text">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>120</y>
      <width>201</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string>Time to have a break</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_move_rest_by_5_min">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Отложить перерыв на 5 мин</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_move_rest_by_10_min">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>320</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Отложить перерыв на 10 мин</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_finish_rest">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>320</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Закончить перерыв</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
